roles/confluent.kerberos/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Install the kerberos packages
+  yum:
+    state: present
+    name: "{{ item }}"
+  with_items: "{{ redhat_packages }}"
+  when: ansible_os_family == "RedHat"
+
+
+- name: Install the kerberos packages
+  apt: name= "{{ item }}" state=present
+  with_items: "{{ ubuntu_packages }}"
+  when: ansible_os_family == "Debian"
+
+- name: Copy the client configuration file
+  template: src=krb5.conf.j2 dest=/etc/krb5.conf
+
+- name: copy Keytabs from ansible host to all hosts
+  synchronize:
+    src: "{{keytab_source_dir}}"
+    dest:  /etc/security/keytabs
```

Comment:
this should be a shared var across all roles, its hardcoded all over the place

Reply:
please file a feature request for this, as this wasn't designed to be configurable in this version of the playbooks.  We can certainly do this in a future release.